python/samples/kernel-syntax-examples/google_search_skill.py

```diff
@@ -0,0 +1,83 @@
+import os
+
+from dotenv import load_dotenv
+
+import semantic_kernel as sk
+from semantic_kernel.connectors.ai.open_ai import OpenAITextCompletion
+from semantic_kernel.connectors.search_engine import GoogleConnector
+from semantic_kernel.core_skills import WebSearchEngineSkill
+
+load_dotenv()
+
+
+async def main():
+    kernel = sk.Kernel()
+    api_key, org_id = sk.openai_settings_from_dot_env()
+    kernel.add_text_completion_service(
+        "dv", OpenAITextCompletion("text-davinci-003", api_key, org_id)
+    )
+
+    """
+    Instantiate a Google Connector
```

Comment:
Can you include in this example a link where people should go to get the API key?